spec.html

```diff
@@ -418,6 +419,28 @@ <h1>CoalesceClassElements ( _elements_ )</h1>
     <emu-note>In the case of public class elements, coalescing corresponds in semantics to ValidateAndApplyPropertyDescriptor. Note that this algorithm only coalesces method and accessor declarations, and it leaves field declarations as is.</emu-note>
   </emu-clause>
 
+  <emu-clause id="sec-to-element-descriptor" aoid="ToElementDescriptor">
+    <h1>ToElementDescriptor ( _field_, _decorators_, _kind_, _enumerable_ )</h1>
+    <emu-alg>
+      1. Assert: _field_ is a <a href="https://tc39.github.io/ecma262/#sec-list-and-record-specification-type">Record</a> as created by <a href="https://tc39.github.io/proposal-private-methods/#runtime-semantics-class-field-definition-evaluation">ClassFieldDefinitionEvaluation</a>.
+      1. Assert: _decorators_ is a List of Decorator or empty List.
+      1. Assert: _kind_ is `"method"` or `"field"`.
+      1. Assert: _enumerable_ is ECMAScript Boolean value.
+      1. let _element_ is newly created ElementDescriptor Record.
+      1. Set _element_.[[Kind]] to _kind_.
+      1. Set _element_.[[Key]] to _field_.[[Name]].
```

Comment:
we can receive descriptor for method that created by abstract operation [ClassElementEvaluation](https://tc39.github.io/proposal-private-methods/#sec-method-definitions-runtime-semantics-classelementevaluation). and, there is `[[Name]]` field. this difference from record that result of [ClassFieldEvaluation](https://tc39.github.io/proposal-private-methods/#runtime-semantics-class-field-definition-evaluation). but in this abstract operation, there is no logic to handle these difference.  So, I think I have to fix this problem when I change pr as review.